plugins/com.google.cloud.tools.eclipse.preferences/schema/com.google.cloud.tools.eclipse.preferences.areas.exsd

```diff
@@ -0,0 +1,178 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Schema file written by PDE -->
+<schema targetNamespace="com.google.cloud.tools.eclipse.preferences" xmlns="http://www.w3.org/2001/XMLSchema">
+<annotation>
+      <appinfo>
+         <meta.schema plugin="com.google.cloud.tools.eclipse.preferences" id="com.google.cloud.tools.eclipse.preferences.areas" name="Preference Areas"/>
```

Comment:
The XML is generated by the Eclipse schema editor.  I'm not going to touch the XML as it will be re-generated another time.